tools/swift-reflection-test/swift-reflection-test.c

```diff
@@ -421,18 +421,21 @@ int doDumpHeapInstance(const char *BinaryFilename) {
             return EXIT_SUCCESS;
           break;
         case Existential: {
+          const char *Name = "_TtP_";
           swift_typeref_t AnyTR
-            = swift_reflection_typeRefForMangledTypeName(RC, "_TtP_", 5);
+            = swift_reflection_typeRefForMangledTypeName(RC,
+              Name, strlen(Name));
 
           printf("Reflecting an existential.\n");
           if (!reflectExistential(RC, Pipe, AnyTR))
             return EXIT_SUCCESS;
           break;
         }
         case ErrorExistential: {
+          const char *ErrorName = "_TtPs5Error_";
           swift_typeref_t ErrorTR
             = swift_reflection_typeRefForMangledTypeName(RC,
-              "_TtPs5Error_", 21);
+              ErrorName, strlen(ErrorName));
```

Comment:
@joewillsher `strlen` is an O(n) operation. Since the string is constant, you can do this to make it a compile-time constant:

```c++
const char ErrorName[] = "_TtPs5Error_";
blahBlah(ErrorName, sizeof(ErrorName) - 1);
```